packages/database/src/services/database.js

```diff
@@ -108,6 +110,16 @@ async function connectToDatabase(dbOptions) {
     logging,
     pool,
   });
+
+  class QueryWithAditConfig extends sequelize.dialect.Query {
+    async run(sql, options) {
+      const userid = namespace.get('audit.userid');
+      if (userid) await super.run(`SELECT set_config('audit.userid', $1, false)`, [userid]);
+      return super.run(sql, options);
+    }
+  }
+  sequelize.dialect.Query = QueryWithAditConfig;
```

Comment:
We should add some kind of comment here to explain what's happening (ie to explain that this is called by sequelize internals with the sql string for every single query done via sequelize).